govc/vm/disk/change.go

```diff
@@ -73,6 +76,7 @@ func (cmd *change) Register(ctx context.Context, f *flag.FlagSet) {
 	f.IntVar(&cmd.key, "disk.key", 0, "Disk unique key")
 	f.StringVar(&cmd.mode, "mode", "", fmt.Sprintf("Disk mode (%s)", strings.Join(vdmTypes, "|")))
 	f.StringVar(&cmd.sharing, "sharing", "", fmt.Sprintf("Sharing (%s)", strings.Join(sharing, "|")))
+	f.Int64Var(&cmd.limit, "disk.io.limit", 0, "Disk storage IO per seconds limit (-1 for unlimited)")
```

Comment:
It's ok to use `0` as default here since this value is not allowed by the API: `A specified parameter was not correct: spec.sharesInfo.limit`